src/fprime_bootstrap/bootstrap_project.py

```diff
@@ -45,14 +45,18 @@ def bootstrap_project(parsed_args: "argparse.Namespace"):
     target_dir = Path(parsed_args.path)
     # Ask user for project name
     project_name = (
-        input(f"Project name ({DEFAULT_PROJECT_NAME}): ") or DEFAULT_PROJECT_NAME
+        (input(f"Project name ({DEFAULT_PROJECT_NAME}): ") or DEFAULT_PROJECT_NAME)
+        if not parsed_args.populate
+        else target_dir.name
     )
     check_project_name(project_name)
 
-    project_path = target_dir / project_name
+    project_path = target_dir / project_name if not parsed_args.populate else target_dir
 
     try:
-        generate_boilerplate_project(project_path, project_name)
+        generate_boilerplate_project(
+            project_path, project_name, populate=parsed_args.populate
+        )
         setup_git_repo(project_path, parsed_args.tag)
```

Comment:
Does this still try to create a git repo?

Reply:
Yes, but it will not error if the .git folder exists.  It just works with it to add the submoudule.